lib/openlogi/shipment.rb

```diff
@@ -0,0 +1,27 @@
+require "openlogi/base_object"
+
+module Openlogi
+  class Shipment < BaseObject
+    attribute :id, String
+    attribute :identifier, String
+    attribute :created_at, DateTime
```

Comment:
Hmm if you look at the sample response for a shipment you will see there is a `created_at` key/value pair. I'm not actually sure what is the definitive source... but you're right that `shipped_at` is also there, so I'll add it.